app/routes/appendix/appendix.js

```diff
@@ -53,7 +53,7 @@ router.use('/', async (req, res, next) => {
 router.route('/')
   .get(async (req, res) => {
     if (!req.templateAppendix.length) {
-      return res.json(req.templateAppendix.view('public'));
+      return res.json(req.templateAppendix.view('extended'));
```

Comment:
We don't want to return extended scope to users, templateId and projectId should be private. Instead, use Sequelize include option to add the template/project object in the return body, see line 52 at app/routes/template/templateAppendix.js

Reply:
I see. Learn it! Added the template and project model to findAll and removed the unnecessary scope.